js/views/TemplateOnly.js

```diff
@@ -0,0 +1,27 @@
+// This is a view that will do nothing more than render
+// a template that it's given when contructed, passing along
+// any context passed into render as context for the template.
+//
+// If you need something beyond that, this view is probably not for you.
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+
+export default class TemplateOnly extends View {
+  constructor(options = {}) {
+    super(options);
+
+    if (!options.template) {
+      throw new Error('Please provide a template');
+    }
+
+    this.options = options;
+  }
+
+  render(context = {}) {
```

Comment:
If you don't pass it in it defaults to empty, but if you pass it in, it will use whatever you pass in.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters